Testing/SystemTests/tests/framework/VesuvioFittingTest.py

```diff
@@ -104,7 +104,8 @@ def runTest(self):
         self.assertTrue(WS_PREFIX + "_NormalisedCovarianceMatrix" in mtd, "Expected covariance workspace in ADS")
 
     def validate(self):
-        self.tolerance = tolerance()
+        # A large tolerance band is required due to differences in the fitting calculations for each platform
+        self.tolerance = 1.2e-2
```

Comment:
Are the differences again just in the calculated errors?

Reply:
When I was manually comparing the two, the only difference was the error bars and I was able to get it to pass on Windows without changing/loosening any of tolerance. However, previously the reference file was generated on Ubuntu, which leads to slightly different values due to some GSL differences (there's a comment in this file citing this). Therefore we require quite a loose tolerance.

Previously the `tolerance()` method would return `~1e-2` for windows, to account for the difference. Now I have the reverse problem, I'm creating the file on Windows, but the tests are running on RHEL, so the tolerance had to change. Its quite an annoying problem as we will just go round in circles of changing the tolerance depending on where we've generated the reference file...

Reply:
Its worse than that. A few years ago 2 different versions of macOS would give slightly different answers, leading to larger tolerances.

Reply:
Thanks for the explanation.
Yes it's been an endless frustration but I still think largely due to the varying versions of GSL. Some future work should hopefully mean we can define a single version across the project and this will get simpler.